opencog/guile/SchemePrimitive.h

```diff
@@ -100,6 +100,7 @@ class SchemePrimitive : public PrimitiveEnviron
 			double (T::*d_hht)(Handle, Handle, Type);
 			double (T::*d_hhtb)(Handle, Handle, Type, bool);
 			double (T::*d_s)(const std::string&);
+			double (T::*d_shhi)(const std::string&, Handle, Handle,int);
```

Comment:
Hi Linas,
 In case you all think it is better to keep handles first convention then
let me know, I will change my code to match the convention.
Regards,
Mandeep

On Fri, Jun 24, 2016 at 9:20 PM, Linas Vepštas <notifications@github.com>
wrote:

> In opencog/guile/SchemePrimitive.h
> <https://github.com/opencog/atomspace/pull/815#discussion_r68396369>:
>
> > @@ -100,6 +100,7 @@ class SchemePrimitive : public PrimitiveEnviron
> >  			double (T::*d_hht)(Handle, Handle, Type);
> >  			double (T::*d_hhtb)(Handle, Handle, Type, bool);
> >  			double (T::*d_s)(const std::string&);
> > +			double (T::*d_shhi)(const std::string&, Handle, Handle,int);
>
> would it be possible to adopt a convention where the handles always come
> first?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/opencog/atomspace/pull/815/files/5699eb5cff6f640287e6a78cb7a7aea66868c37f#r68396369>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHM_f-Xex4tQ33tLTX9a7FLFdkjFR6s9ks5qO9mZgaJpZM4I9dhG>
> .
>